config/application.rb

```diff
@@ -24,5 +24,7 @@ class Application < Rails::Application
     end
 
     config.middleware.use Rack::Attack
+
+    config.autoload_paths += %W(#{config.root}/app/models/comments)
```

Comment:
Nie mogę się nadziwić, że ta linia jest potrzebna - wszystkie podkatalogi `app/models` powinny już być w autoloadzie. Czy mógłbyś jeszcze raz to sprawdzić i się upewnić?